common/src/main/java/com/hedera/block/common/utils/Preconditions.java

```diff
@@ -27,11 +27,11 @@ public final class Preconditions { // @todo(381) change the APIs to accept non-n
      * we return it, else we throw {@link IllegalArgumentException}.
      *
      * @param toCheck a {@link String} to be checked if is blank as defined
-     * above
+     *                above
```

Comment:
If you're using `spotlessApply` we should see these type of reformatting changes on files we touch since Jasper and Joseph changed the formatting profile